RevenueCatUI/CustomerCenter/Data/CustomerCenterPurchases.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  CustomerCenterPurchases.swift
+//
+//  Created by Cesar de la Vega on 18/7/24.
+
+import Foundation
+import RevenueCat
+
+@available(iOS 15.0, macOS 12.0, tvOS 15.0, *)
```

Comment:
Probably doesn't matter right now since we only support iOS, but I think this line isn't needed with the line below?